database/migrations/2023_08_10_094036_add_logo_image_id_to_clinics.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+    Schema::table('clinics', function (Blueprint $table) {
+            $table->unsignedBigInteger('logo_image_id')->nullable();
+            $table->foreign('logo_image_id')->references('id')->on('logo_images')->onDelete('set null');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::table('clinics', function (Blueprint $table) {
+            //
```

Comment:
fali micanje što si napravio u up() znači 
dropColumn logo_image_id
i uklanjanje reference